app/models/calculation_result.rb

```diff
@@ -20,30 +20,95 @@ def income_contribution
     monetise(api_response.dig(:result_summary, :overall_result, :income_contribution))
   end
 
+  def gross_income
+    monetise(api_response.dig(:result_summary, :gross_income, :total_gross_income))
+  end
+
+  def gross_outgoings
+    monetise(api_response.dig(:result_summary, :disposable_income, :total_outgoings_and_allowances))
+  end
+
+  def gross_income_limit
+    monetise(api_response.dig(:result_summary, :gross_income, :proceeding_types).map { |pt| pt.fetch(:upper_threshold) }.min)
+  end
+
+  def total_disposable_income
+    monetise(api_response.dig(:result_summary, :disposable_income, :total_disposable_income))
+  end
+
+  def disposable_income_limit
+    monetise(api_response.dig(:result_summary, :disposable_income, :proceeding_types).map { |pt| pt.fetch(:upper_threshold) }.min)
+  end
+
+  def total_capital
+    monetise(api_response.dig(:result_summary, :capital, :total_capital))
+  end
+
+  def total_capital_limit
+    monetise(api_response.dig(:result_summary, :capital, :proceeding_types).map { |pt| pt.fetch(:upper_threshold) }.min)
+  end
+
   def client_income_rows
-    {
-      employment_income: extract_money(:result_summary, :disposable_income, :employment_income, :gross_income),
-      benefits: extract_money(:assessment, :gross_income, :state_benefits, :monthly_equivalents, :all_sources),
+    data = {
+      employment_income: api_response.dig(:result_summary, :disposable_income, :employment_income, :net_employment_income),
```

Comment:
In UAT of #81  Mike Hooper asked me to change from net_employment_income to gross_income for this field, because the `net` value CFE returns has removed not just NI and tax but also the £45 employment disregard, so shows a very user-unfriendly number if not explained further.